package.json

```diff
@@ -57,6 +57,7 @@
     "mongoose": "8.0.3",
     "multer": "^1.4.5-lts.1",
     "nestjs-real-ip": "3.0.1",
+    "openapi3-ts": "^4.2.1",
```

Comment:
?

Reply:
yarn installed peer dependencies automatically, with pnpm I need to do it manually

Reply:
openapi3-ts isn't used but in 1 spot?

Reply:
Yeah, only in one spot...